workflow/cron/controller.go

```diff
@@ -126,21 +126,19 @@ func (cc *Controller) processNextCronItem() bool {
 		return false
 	}
 	defer cc.cronWfQueue.Done(key)
-	logCtx := log.WithField("key", key)
+	logCtx := log.WithField("cronworkflow", key)
```

Comment:
revert?

Reply:
We need to distinct  the CronWorkflow controller logs from workflow controller. This `key` is Cronworkflow Name.

Reply:
ok, can it be `cronWorkflow`?

Reply:
capital W